syntax = "proto3";

package PbCan;

message CanMessage
{
	uint64 id = 1;
	string name = 2;
	uint64 data = 3; 
	uint64 timestamp = 4;
	uint64 frequency = 5;
}

message CanNetwork
{
	string name = 1;
	repeated CanMessage canMessages = 2;
}

message CanNetworks
{
	uint64 timestamp = 1;
	repeated CanNetwork networks = 2;
}